### DISCUSSION TITLE
运行GPT2案例出现RuntimeError: Could not find 'SLURM_PROCID'问题，是必须要装SLURM环境？

### DISCUSSION THREAD

**Discussion by ZXM1063694570** [hpcaitech/ColossalAI#1371] [Community | Q&A]
使用了提供的Dockerhub上的镜像0.1.7，但是在运行GPT案例时候出现RuntimeError: Could not find 'SLURM_PROCID'问题，并且在0.1.8镜像版本中也是如此
![image](https://user-images.githubusercontent.com/65949265/181002381-29631a46-651b-4495-8c9e-8a4b8b7e65ef.png)
![image](https://user-images.githubusercontent.com/65949265/181002456-3f17e423-6d20-4cce-b54c-16e797464acd.png)
这是我的run脚本：
![image](https://user-images.githubusercontent.com/65949265/181002522-02a0d5dd-d77b-4903-905b-a8da1da8a69e.png)
其中我的gpt2_configs配置换了其他的配置也出现同样的问题

Environment：
docker pull hpcaitech/colossalai:0.1.7 & 0.1.8
pip install transformers
pip install titans

8张A100

**Comment by YuliangLiu0306** [comment]
Adding '--from_torch' after train_gpt.py would be helpful.
